examples/kyb-app/src/components/organisms/UIRenderer/utils/generateBlocks.tsx

```diff
@@ -0,0 +1,37 @@
+//@ts-nocheck
+import { UIElement } from '@app/domains/collection-flow';
+import { ElementsMap } from '../types/elements.types';
+import { createBlocks } from '@ballerine/blocks';
+import { v4 } from 'uuid';
+
+export const generateBlocks = (schema: UIElement | UIElement[], elements: ElementsMap) => {
+  let base = createBlocks<keyof typeof elements>().addBlock();
```

Comment:
The generic passed here is required to be a discriminated union where every union member has `type` set to a string literal. Note the blocks declare module step was also skipped.